app/models/course/lesson_plan/item.rb

```diff
@@ -21,6 +21,10 @@ class Course::LessonPlan::Item < ApplicationRecord
     order(:start_at, :title)
   end)
 
+  scope :with_components_enabled, lambda { |actable_types|
```

Comment:
We should rename this to reflect the behavior of the scope. Also, can add some documentation on this as well?